extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -38,6 +40,8 @@ function TrackedCornerstoneViewport({
   dataSource,
   displaySet,
   viewportIndex,
+  servicesManager,
```

Comment:
is this actually receiving the `servicesManager`? If it is, it likely wouldn't need the `ToolBarService`

Reply:
Nope, forgot to remove this prop, i'm passing the toolbar service, thanks